.github/workflows/package_constraint_solver.yaml

```diff
@@ -1,9 +1,13 @@
 name: Package constraint_solver tests
 on:
   push:
-    branches: [ "main" ]
+    branches:
+      - main
+      - master
```

Comment:
`master` should be a locked copy of `main` that explicitly blocks pushes

Reply:
You can even drop the `master` branch once `main` is added, since GitHub [redirects](https://github.com/github/renaming#whats-already-changed) "links that contain the deleted branch name such as `master` to the corresponding link in the repository's new default branch."

Reply:
Ooooh shiny

Reply:
The wrinkle being... `dart-lang/samples` doesn't (yet?) have a `main` branch.

Reply:
What are you suggesting here? My understanding is that we should be switching from `master` to `main` if we can.

Reply:
I went ahead and changed `master` to `main`.